doc/classes/GraphEdit.xml

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GraphEdit" inherits="Control" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		GraphEdit is a control responsible for displaying and manipulating graph-like data using [GraphNode]s. It provides access to creation, removal, connection, and disconnection of nodes.
+		An editor for graph-like structures, using [GraphNode]s.
```

Comment:
Word "editor" might suggest that it can be used only for plugins, idk.

Reply:
Haven't changed this yet. I don't think we should restrict ourselves from describing editors as such.